src/makeInfinityAdder.js

```diff
@@ -4,7 +4,21 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sumOfNumbers = 0;
+
+  return function adder(number = NaN) {
+    if (Number.isNaN(number)) {
```

Comment:
The use of `Number.isNaN(number)` is appropriate here to check if the passed argument is `NaN`. This ensures that the function behaves correctly when a non-number is passed.